### ISSUE TITLE
Incompatible with prometheus-client 0.14.0

### ISSUE THREAD

**Issue by kojiromike** [rycus86/prometheus_flask_exporter#125]
in the last hour I started getting this error:

```
ImportError: cannot import name 'choose_encoder' from 'prometheus_client.exposition'
```

because prometheus-client 0.14.0 was released and [doesn't have the `choose_encoder` function anymore](https://github.com/prometheus/client_python/compare/v0.13.1...v0.14.0#diff-31616dde80dc7c3540794d368828f5fdf09536ba95ceb8fef6463e76c35c97e6L215).

**Comment by rycus86** [comment]
This is now fixed by @CorinneKelly - thanks a lot! (see https://github.com/rycus86/prometheus_flask_exporter/pull/126)
I've published the new version as `0.20.0` - which also drops support for Python 2 and Python 3.5.

**Comment by rycus86** [comment]
Thanks to @kojiromike adding a change in https://github.com/rycus86/prometheus_flask_exporter/pull/129 the `0.20.1` version should now work with the latest and also with the older version of `prometheus_client`, so pinning either of them or both should all work. (for example if Renovate is bumping your packages one by one).
